tests/console/openldap2.pm

```diff
@@ -25,6 +25,9 @@ my @tests_sle = '';
 sub install_openldap2 {
     record_info 'Install OpenLDAP 2';
     zypper_call('in openldap2-*');
+
+    record_info 'Install bzip2 if missing';
```

Comment:
just install bzip2 with openldap in previous zypper_call, you don't need to record info about it